src/OutputWriters/jld2_output_writer.jl

```diff
@@ -11,11 +11,12 @@ default_included_properties(::ShallowWaterModel) = [:grid, :coriolis, :closure]
 default_included_properties(::HydrostaticFreeSurfaceModel) = [:grid, :coriolis, :buoyancy, :closure]
 
 """
-    JLD2OutputWriter{I, T, O, IF, IN, KW} <: AbstractOutputWriter
```

Comment:
I was first confused what `I` was but I now see this is just a comment.  Thanks for updating it.